css/apps/media.scss

```diff
@@ -273,4 +273,33 @@
 			}
 		}
 	}
+
+	.file-picker__actions {
```

Comment:
please write a comment to which modal window this styles are applied

Reply:
Done